src/Components/Components/src/Auth/AuthenticationState.cs

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Security.Claims;
+
+namespace Microsoft.AspNetCore.Components
+{
+    /// <summary>
+    /// Provides information about the currently authenticated user, if any.
+    /// </summary>
+    public class AuthenticationState
+    {
+        /// <summary>
+        /// Gets a <see cref="ClaimsPrincipal"/> that describes the current user.
+        /// </summary>
+        public ClaimsPrincipal User { get; }
```

Comment:
nit: fields -> constructors -> properties

Reply:
OK. Rather than go through another multi-hour CI process to make this tweak, I'm going to include it in my next auth PR.